package.json

```diff
@@ -3,10 +3,12 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "generate": "true",
-    "format": "prettier --write --ignore-path .gitignore ."
+    "generate": "./scripts/generate.sh && npm run format",
+    "format": "poetry run black ."
```

Comment:
```suggestion
    "format": "prettier --write --ignore-path .gitignore .",
    "preformat": "make format"
```

Reply:
Why do we need prettier here?

Reply:
https://github.com/seamapi/python-next/pull/6/commits/850fb9a4767781c4f2eac476b9ff374bcd9bac2c

Reply:
For the rest of the project, e.g., package.json, README, etc.